wdfn-server/waterdata/templates/wdfn.html

```diff
@@ -174,6 +174,14 @@ <h4>
               </div>
             </form>
         </div>
-      {{ USMapComponent(latitude, longitude, zoom) }}
+        <div class="wdfn-component wdfn-map-container" 
```

Comment:
![the end result](https://user-images.githubusercontent.com/6290/91490187-0aa0f500-e880-11ea-92fe-457a88f5947d.png)